src/AlternateLife.RageMP.Net/Data/EntityData.cs

```diff
@@ -1,8 +1,6 @@
 using System;
 using System.Runtime.InteropServices;
-using AlternateLife.RageMP.Net.Enums;
```

Comment:
Avoid changing namespace directives if you don't need to edit this file anyways.